cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/InitCommandTest.java

```diff
@@ -93,15 +91,53 @@ public void testInitCommandWithMain() throws IOException {
         String tomlContent = Files.readString(
                 packageDir.resolve(ProjectConstants.BALLERINA_TOML), StandardCharsets.UTF_8);
         String expectedContent = "[build-options]\n" +
-                "observabilityIncluded = true\n";
-        Assert.assertEquals(tomlContent, expectedContent);
+                "observabilityIncluded = true";
+        Assert.assertTrue(tomlContent.contains(expectedContent));
 
         Assert.assertTrue(Files.exists(packageDir.resolve("main.bal")));
-        Assert.assertTrue(Files.notExists(packageDir.resolve(ProjectConstants.PACKAGE_MD_FILE_NAME)));
+        Assert.assertTrue(Files.exists(packageDir.resolve(ProjectConstants.TEST_DIR_NAME)));
+        Assert.assertTrue(Files.exists(packageDir.resolve(ProjectConstants.PACKAGE_MD_FILE_NAME)));
 
         Assert.assertTrue(readOutput().contains("Created new Ballerina package"));
     }
 
+    @Test(description = "Test init command with general template")
+    public void testInitCommandWithGeneral() throws IOException {
+        // Test if no arguments was passed in
+        Path packageDir = tmpDir.resolve("sample2");
+        Files.createDirectory(packageDir);
+        String[] args = {"myproject", "-t", "general"};
```

Comment:
The user should not be able to give this template. This is only for internal classification. We should only allow giving `main`, `service` or `lib` no nothing at all.